packages/calling-component-bindings/src/utils/participantListSelectorUtils.ts

```diff
@@ -189,6 +219,33 @@ const convertRemoteParticipantToParticipantListParticipantBetaRelease = (
 
 /* @conditional-compile-remove(reaction) */
 const convertRemoteParticipantToParticipantListParticipantBeta = (
+  userId: string,
+  displayName: string | undefined,
+  state: RemoteParticipantState,
+  isMuted: boolean,
+  isScreenSharing: boolean,
+  isSpeaking: boolean,
+  raisedHand: RaisedHandState | undefined,
+  localUserCanRemoveOthers: boolean,
+  reaction: Reaction | undefined
+): CallParticipantListParticipant => {
+  return {
+    ...convertRemoteParticipantToParticipantListParticipant(
+      userId,
+      displayName,
+      state,
+      isMuted,
+      isScreenSharing,
+      isSpeaking,
+      raisedHand,
+      localUserCanRemoveOthers
+    ),
+    reaction
+  };
+};
+
+/* @conditional-compile-remove(spotlight) */
+const convertRemoteParticipantToParticipantListParticipantBetaSpotlight = (
```

Comment:
These functions are really confusing - why do we need individual ones for different features/beta?
Is conditional compile not working on the arguments?

Reply:
If not @PorterNan is there another value we need to enable in the babel config?

Reply:
When trying to build in the beta-release flavor for any other feature than Spotlight, it would remove the import for spotlight but these functions would have params that require spotlight object/types. So was facing build issues and having functions separate fixes them.

Reply:
@JamesBurnside yes conditional compile does not work on the function arguments so we need a separate function for each feature arg.

Reply:
Could we still pass in the argument but make it `undefined` in stable builds?

Reply:
We could do that. But that does not solve the problem where imports are stripped in stable builds but the params of the functions still wants those beta imports.